autopts/ptsprojects/boards/nrf53.py

```diff
@@ -30,22 +32,17 @@ def build_and_flash(zephyr_wd, board, debugger_snr, conf_file=None, *args):
                   board, conf_file)
 
     tester_dir = os.path.join(zephyr_wd, 'tests', 'bluetooth', 'tester')
-    controller_dir = os.path.join(zephyr_wd, 'samples', 'bluetooth', 'hci_ipc')
 
     check_call('rm -rf build/'.split(), cwd=tester_dir)
-    check_call('rm -rf build/'.split(), cwd=controller_dir)
 
     bttester_overlay = 'hci_ipc.conf'
 
     if conf_file and conf_file != 'default' and conf_file != 'prj.conf':
         bttester_overlay += f';{conf_file}'
 
-    cmd = ['west', 'build', '--no-sysbuild', '-b', board, '--', f'-DEXTRA_CONF_FILE=\'{bttester_overlay}\'']
+    cmd = ['west', 'build', '--sysbuild', '-b', board, '--', f'-DEXTRA_CONF_FILE=\'{bttester_overlay}\'']
     check_call(cmd, cwd=tester_dir)
-    check_call(['west', 'flash', '--skip-rebuild', '--recover', '-i', debugger_snr], cwd=tester_dir)
-
-    cmd = ['west', 'build', '--no-sysbuild', '-b', 'nrf5340dk/nrf5340/cpunet', '--',
-           f'-DEXTRA_CONF_FILE=\'nrf5340_cpunet_iso-bt_ll_sw_split.conf;'
-           f'../../../tests/bluetooth/tester/hci_ipc_cpunet.conf\'']
-    check_call(cmd, cwd=controller_dir)
-    check_call(['west', 'flash', '--skip-rebuild', '-i', debugger_snr], cwd=controller_dir)
+    try:
+        check_call(['west', 'flash', '--skip-rebuild', '-i', debugger_snr], cwd=tester_dir)
+    except CalledProcessError:
+        check_call(['west', 'flash', '--skip-rebuild', '--recover', '-i', debugger_snr], cwd=tester_dir)
```

Comment:
just wonder,  does recovering take significant amount of time?

Reply:
This command without recover takes about 39.5 seconds on my machine. With recover that's 42.5 seconds, so it adds about 3 seconds on my machine. 

I'm OK keeping it simpler and just always do recover if you prefer that. Copied this from the 54 series files (not sure if it faster/slower on that platform)

Reply:
nah,  this is fine I guess